docs/pages/eas-update/request-proxying.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Request Proxying
+description: Proxy requests to the EAS Update server through your own server.
```

Comment:
```suggestion
description: Proxy requests to the EAS Update server through your own server.
hideTOC: true
```

Let's the hide TOC on this page since it contains only one section.